vars/jira.groovy

```diff
@@ -1,35 +1,52 @@
 #!/usr/bin/env groovy
 
+def lastSuccessfulBuild(passedBuilds, build) {
+    if ((build != null) && (build.result != 'SUCCESS')) {
+        echo "Finding last succesful build"
+        passedBuilds.add(build)
+        echo "Found a build"
+        lastSuccessfulBuild(passedBuilds, build.getPreviousBuild())
+    }
+}
+
 @NonCPS
-def getChangeString() {
-    MAX_MSG_LEN = 100
+def getChangeString(passedBuilds) {
```

Comment:
if this function is not meant to be called by pipeline, maybe we can use some naming convention to indicate private API (eg. `_getChangeString`)